package.json

```
@@ -11,6 +11,7 @@
     "lint": "standard rollup.config.js index.es.js",
     "fix": "standard --fix rollup.config.js index.es.js",
     "test": "cd test && rm -f output.* && rollup -c && cmp output.js expected.js && cmp output.css expected.css && cd ..",
+    "test-sass": "cd test && rm -f output.* && rollup -c rollup.config.sass.js && cmp output.js expected.js && cmp output.css expected.css && cd ..",
```

Move each runtime test to its own directory under tests.
```suggestion
    "test": "cd test/node-sass && rm -f output.* && rollup -c && cmp output.js expected.js && cmp output.css expected.css && cd ..",
    "test-sass": "cd test/sass && rm -f output.* && rollup -c rollup.config.js && cmp output.js expected.js && cmp output.css expected.css && cd ..",
```

---

I have implemented your suggestions, thank you.
I have to find how to update the merge request with a new commit :)